src/controller/CHIPDeviceController.cpp

```diff
@@ -1960,6 +1982,7 @@ void DeviceCommissioner::OnDeviceConnectedFn(void * context, Messaging::Exchange
 void DeviceCommissioner::OnDeviceConnectionFailureFn(void * context, const ScopedNodeId & peerId, CHIP_ERROR error)
 {
     // CASE session establishment failed.
+    MATTER_LOG_METRIC_END(kMetricDeviceOperationalSetup, error);
```

Comment:
This seems like a very generic metric name, for a place which is not involved in most "operational setup". This bit is specific to operational session setup at the end of commissioning.